airflow/operators/email.py

```diff
@@ -1,91 +0,0 @@
-#
```

Comment:
Can't remember now - I think we agreed that we will do "redirection" of all such moved providers - but I totally don't remember what we settled on  @Lee-W ?

Reply:
I remember you said we're going to do that, but I was informed instead of being part of the discussion 🤔

Currently, we still suggest users to use the provider in ruff thought